File: UaRestClient/Model/Measurements/Constants/TypeScript/measurements_constants.ts
export const NS = 'urn:opcfoundation.org:2024-10:starterkit:measurements';

export class BrowseNames {
   static readonly MeasurementContainerType: string = 'MeasurementContainerType'
   static readonly Orientation: string = 'Orientation'
   static readonly OrientationDataType: string = 'OrientationDataType'
   static readonly Pressure: string = 'Pressure'
   static readonly Reset: string = 'Reset'
   static readonly Temperature: string = 'Temperature'
}

export class DataTypeIds {
    static readonly OrientationDataType: string = 'nsu=' + NS + ';i=31'
}

export class MethodIds {
    static readonly MeasurementContainerType_Reset: string = 'nsu=' + NS + ';i=24'
}

export class ObjectIds {
    static readonly OrientationDataType_Encoding_DefaultBinary: string = 'nsu=' + NS + ';i=32'
    static readonly OrientationDataType_Encoding_DefaultXml: string = 'nsu=' + NS + ';i=40'
    static readonly OrientationDataType_Encoding_DefaultJson: string = 'nsu=' + NS + ';i=48'
}

export class ObjectTypeIds {
    static readonly MeasurementContainerType: string = 'nsu=' + NS + ';i=1'
}

export class VariableIds {
    static readonly MeasurementContainerType_Temperature: string = 'nsu=' + NS + ';i=2'
    static readonly MeasurementContainerType_Temperature_EngineeringUnits: string = 'nsu=' + NS + ';i=7'
    static readonly MeasurementContainerType_Temperature_EngineeringUnits_EngineeringUnits: string = 'nsu=' + NS + ';i=12'
    static readonly MeasurementContainerType_Pressure: string = 'nsu=' + NS + ';i=13'
    static readonly MeasurementContainerType_Pressure_EngineeringUnits: string = 'nsu=' + NS + ';i=18'
    static readonly MeasurementContainerType_Pressure_EngineeringUnits_EngineeringUnits: string = 'nsu=' + NS + ';i=23'
    static readonly MeasurementContainerType_Reset_InputArguments: string = 'nsu=' + NS + ';i=25'
    static readonly MeasurementContainerType_Reset_OutputArguments: string = 'nsu=' + NS + ';i=26'
    static readonly MeasurementContainerType_Orientation: string = 'nsu=' + NS + ';i=27'
}

File: UaRestClient/Model/Measurements/Constants/Python/measurements_constants.py
from enum import Enum

class Namespaces(Enum):
     Uri = "urn:opcfoundation.org:2024-10:starterkit:measurements"

class BrowseNames(Enum):
    MeasurementContainerType = "MeasurementContainerType"
    Orientation = "Orientation"
    OrientationDataType = "OrientationDataType"
    Pressure = "Pressure"
    Reset = "Reset"
    Temperature = "Temperature"

class DataTypeIds(Enum):
    OrientationDataType = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=31"

def get_DataTypeIds_name(value: str) -> str:
    try:
        return DataTypeIds(value).name
    except ValueError:
        return None


class MethodIds(Enum):
    MeasurementContainerType_Reset = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=24"

def get_MethodIds_name(value: str) -> str:
    try:
        return MethodIds(value).name
    except ValueError:
        return None


class ObjectIds(Enum):
    OrientationDataType_Encoding_DefaultBinary = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=32"
    OrientationDataType_Encoding_DefaultXml = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=40"
    OrientationDataType_Encoding_DefaultJson = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=48"

def get_ObjectIds_name(value: str) -> str:
    try:
        return ObjectIds(value).name
    except ValueError:
        return None


class ObjectTypeIds(Enum):
    MeasurementContainerType = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=1"

def get_ObjectTypeIds_name(value: str) -> str:
    try:
        return ObjectTypeIds(value).name
    except ValueError:
        return None


class VariableIds(Enum):
    MeasurementContainerType_Temperature = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=2"
    MeasurementContainerType_Temperature_EngineeringUnits = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=7"
    MeasurementContainerType_Temperature_EngineeringUnits_EngineeringUnits = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=12"
    MeasurementContainerType_Pressure = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=13"
    MeasurementContainerType_Pressure_EngineeringUnits = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=18"
    MeasurementContainerType_Pressure_EngineeringUnits_EngineeringUnits = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=23"
    MeasurementContainerType_Reset_InputArguments = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=25"
    MeasurementContainerType_Reset_OutputArguments = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=26"
    MeasurementContainerType_Orientation = "nsu=urn:opcfoundation.org:2024-10:starterkit:measurements;i=27"

def get_VariableIds_name(value: str) -> str:
    try:
        return VariableIds(value).name
    except ValueError:
        return None

